docs/languages/java/building_codemod.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 4
+title: Building & Running Your Codemod
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+There are 3 steps for turning our codemod type into a runnable codemod artifact -- creating the CLI entry point, pointing to that entry point!
+
+## Create the CLI Entry Point
+
+Codemodder is a library for building codemod CLIs... so where is the CLI part? We need to create a small piece of code to actually invoke our codemods. Let's create a new type, `App.java`:
```

Comment:
```suggestion
Codemodder is a library for building codemods. We need to write a small piece of code to enable our codemods to be invoked using a CLI entry point. Let's create a new type, `App.java`:
```

Reply:
Just for the record: this is not exactly how we have conceived of third-party codemod development in Python. Codemodder is implemented as a library _and_ a CLI and we sort of anticipated using a plugin architecture to enable third-party codemods to run as well. Probably this is language-specific enough that if we document it well the difference  between Python and Java isn't really important.